tests/mocks/constants.py

```diff
@@ -0,0 +1,69 @@
+import json
+from typing import TypedDict
+import os
+
+mock_dir = os.path.dirname(__file__)
+
+mock_client_keys_file = open(os.path.join(mock_dir, "mockClientKeys.json"))
+mock_private_key_pkcs1 = open(os.path.join(mock_dir, "mockPrivateKeyPkcs1.pem"))
+mock_private_key_pkcs8 = open(os.path.join(mock_dir, "mockPrivateKeyPkcs8.pem"))
+mock_jwks_file = open(os.path.join(mock_dir, "mockPublicJwks.json"))
+mock_alternate_jwks_file = open(os.path.join(mock_dir, "mockPublicJwksAlternate.json"))
+
+mock_client_keys = json.load(mock_client_keys_file)
+mock_jwks = json.load(mock_jwks_file)
+mock_jwks_alternate = json.load(mock_alternate_jwks_file)
+
+
+hostname = "https://id.sgid.com"
+
+
+class MockConstants(TypedDict):
+    server: dict[str, str]
+    client: dict[str, str]
+    data: dict[str, any]
+
+
+MOCK_CONSTANTS: MockConstants = {
+    "server": {
+        # private_key and public_jwks are a matching pair
+        "private_key": mock_private_key_pkcs1.read(),
+        "public_jwks": mock_jwks,
+        # public jwks which does not correspond to any private key
+        "public_jwks_alternate": mock_jwks_alternate,
+        "hostname": hostname,
+        "auth_endpoint": f"{hostname}/v2/oauth/authorize",
```

Comment:
minor suggestion: can consider move /v2 to a variable so that in the future, we just need to change one variable instead of individual routes.

Reply:
good point! 0f78d2c